transformer_engine/pytorch/attention.py

```diff
@@ -429,8 +431,9 @@ class FlashAttnUnpaddedFuncWithCP(torch.autograd.Function):
     """
 
     @staticmethod
-    def forward(ctx, q, k, v, cu_seqlens_q, cu_seqlens_k, max_seqlen_q, max_seqlen_k, dropout_p,
-                cp_group, cp_global_ranks, cp_stream, softmax_scale, causal, deterministic):
+    def forward(ctx, is_training, q, k, v, cu_seqlens_q, cu_seqlens_k, max_seqlen_q, max_seqlen_k,
```

Comment:
Is it still necessary to call this function `FlashAttnUnpaddedFuncWithCP` since both `FlashAttention` and `FusedAttention` are supported now? Maybe `AttnUnpaddedFuncWithCP`? What does the `Unpadded` here mean? Does it mean that the mask type can't be `padding`?

Reply:
If has unpadded becaused FA v1 has function name like `flash_attn_unpadded_func`, but now we are not limited to   v1 any more. I think we can have a better naming. As you suggested, can we change it to `AttnFuncWithCP`?

Reply:
Yes, that works!

Reply:
Can you remove the `flash_` in `flash_attn_forward_func_with_cp` as well? Thanks.